README.md

```diff
@@ -159,6 +180,95 @@ ember exam --split=3 --partition=3
+
+For example, if you specify load-balance equals 3, then three browser instances will be created and the output will look something like::
+
+```bash
+ok 1 Chrome 66.0 - Browser Id 1 - some test
+ok 2 Chrome 66.0 - Browser Id 2 - some another test
+ok 3 Chrome 66.0 - Browser Id 3 - some the other test
+```
+
+You can also specify the `split` and `partition` options with `load-balance` to load a portion of test modules on multiple CI containers.
+# TODO - add description how `split` options is different from one without `load-balance`
+
+```bash
+$ ember exam --split=<num> --partition=<num> --load-balance=<num>
+```
+
+This command will split test files and load-balance tests from the specified partition across the browsers. For example --split=2 -partition=1 --load-balance=3, the complete list of test files are split into two halves. With the first half of the list load balanced against three browsers. The output will look something like below:
+```base
+ok 1 Chrome 66.0 - Exam Partition 1 - browser Id 1 - some test
+ok 2 Chrome 66.0 - Exam Partition 1 - browser Id 2 - another test
+ok 3 Chrome 66.0 - Exam Partition 1 - browser Id 3 - some the other test
+```
+
+If `load-balance` is used without specifying a number of browser(s), the option has a default value of 1 and must be passed last. Otherwise, any following arguments will be read as a value for `--load-balance`. For instance:
+
+```bash
+# don’t do this
+# The number of browser instances = --server --no-launch
+$ ember exam --load-balance --server --no-launch
+
+# do this
+$ ember exam --server --no-launch --load-balance
+```
+
+Due to the dynamic nature of the `load-balance` option, test file execution order could vary between runs. To reproduce a past test execution follow the next section.
+
+**Note 1** _You must be using ember-cli version 3.2.0 or greater for this feature to work properly._
+**Note 2** _You must be using ember-qunit version 4.1.1 or greater for this feature to work properly._
+**Note 3** _You must be using qunit version 2.8.0 or greater for this feature to work properly._
+**Note 4** _This feature is not currently supported by Mocha._
+
+##### Test Failure Reproduction
+
+Since Ember Exam with `load-balance` loads modules one at a time the order of executing modules is not deterministic. For this reason a test execution with `load-balance` creates a json file to track which browser executes which modules in which order and the file is stored in the root directory and the naming structure is `test-execution-YYYY-MM-DD--HH-MM-SS.json`.
+To replay the test execution for one particular browser, do the following:
+
+```bash
+$ ember exam --replay-execution=[string] --replay-browser=[num]
+```
+
+`replay-execution` allows you to specify a path to the json file to run execution against and `replay-browser` is to specify browser IDs to execute.
+
+```bash
+$ ember exam --replay-execution=test-execution-000000.json --replay-browser=1
```

Comment:
can one provide `--replay-browser=1,2,3` to specify more then 1 browser (but not all) to replay?

If not, that is fine, it could easily be described as out-of-scope of an initial implementation.